pkg/process/monitor/process_monitor.go

```diff
@@ -0,0 +1,247 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
documentation in the file (structs, fields in structs, functions) can help other people to better understand the code and troubleshoot / add features in the future.

Reply:
done